packaging/Jenkinsfile

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/groovy
```

Comment:
This file should be removed, it's only used in the packaging repo to test it.

Reply:
Ok, removing